packages/@aws-cdk/cfnspec/build-tools/update-metrics.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+explorer_tarball="$1"
+
+target_file=lib/canned-metrics/services.json
+
+mkdir -p $(dirname $target_file)
+
+tar xzOf "$explorer_tarball" "package/dist/lib/configuration/generated/services.json" > $target_file
```

Comment:
The `services.json` file is now "prettified", whereas I think it wasn't initially. Can the command to prettify it go in this script, so we're consistent in how we're doing it?

Reply:
Good catch